src/mds/MDSRank.cc

```diff
@@ -3106,6 +3106,9 @@ void MDSRank::command_dump_tree(const cmdmap_t &cmdmap, std::ostream &ss, Format
   std::string root;
   int64_t depth;
   cmd_getval(cmdmap, "root", root);
+  if (root.empty()) {
+    root = "/";
+  }
```

Comment:
Add a trivial unit test in `qa/tasks/cephfs/test_admin.py` for this.

Reply:
sure.